app/src/main/java/org/systers/mentorship/viewmodels/RequestDetailViewModel.kt

```diff
@@ -82,8 +81,7 @@ class RequestDetailViewModel : ViewModel() {
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribeWith(object : DisposableObserver<CustomResponse>() {
                     override fun onNext(customResponse: CustomResponse) {
-                        message = customResponse.message ?: MentorshipApplication.getContext()
-                                .getString(R.string.error_something_went_wrong)
+                        message = customResponse.message
```

Comment:
Are you sure `customResponse.message` will never be null?

Reply:
Yes, because `onNext()` is only called if there is a successful response. And for every successful response, we are sending a `CustomResponse` which will have a message field.